test/e2e/networks/byzantine.toml

```diff
@@ -0,0 +1,7 @@
+[node.validator01]
```

Comment:
I suggest we just don't commit this network, seems like it's mostly been used for testing during development.

Reply:
ok